source/index.html.md.erb

```diff
@@ -1,25 +1,67 @@
 ---
-title: About the [name of API] API
+title: EMCS API testing guide
 weight: 10
 ---
 
-# About the [name of API] API
+# EMCS API testing guide
 
-<%= warning_text("You can use the Technical Documentation Template (TDT) to create and publish technical documentation for users in government, including API documentation.<br/><br/>
-We’ve provided these starting pages with a best practice API documentation structure. It includes the template structure of the documentation sections you need, and is based on <a href='https://www.gov.uk/guidance/how-to-document-apis'>our latest guidance on how to document APIs</a>.<br/><br/>
-You can also start from scratch if you’re not using the TDT to create API documentation.") %>
+This is a prototype testing guide used for research. The testing process it describes is still in development.
 
-Use this section to start your documentation with a short overview of:
+Learn how to test the compatibility of your software with [Excise Movement Control System (EMCS) API](api-documentation/docs/api/service/excise-movement-control-system-api/1.0).
 
-- what your API does and does not do
-- whether your API is in the alpha, beta or live phase
-- the benefits of using the API, so users can compare it to other APIs
-- whether it’s free to use
-- whether your users need to build or buy anything before they use it (technical prerequisites)
-- who can use your API, and any restrictions on using it or its data
+## Before you start
 
-Avoid trying to sell your API. Describe features as benefits for your users instead.
+When you are ready to test your software, first read and understand the [EMCS API service guide](guides/emcs-api-service-guide/) and [EMCS API technical interface specification]() if you have not already done so.
```

Comment:
For URL links we will need the preceding `/` e.g. `/guides/emcs-api-service-guide/` This seems to be missing here.